docs/translations/translations-in-FSE-templates.md

```diff
@@ -0,0 +1,121 @@
+# Translations in FSE templates
+
+To make the WooCommerce Blocks plugin inclusive, all user-facing strings should be translatable. Manaing [Translations in PHP files](docs/translations/translations-in-PHP-files.md) and [Translations in JS/TS files](docs/translations/translations-in-JS-TS-files.md) is rather simple, given that both languages are programming languages. As FSE-templates are using plain HTML, translating them works slightly different.
```

Comment:
> given that both languages are programming languages.

Could you explain this a little more, why does them being programming languages affect the simplicity of managing translations?

Reply:
Done! I explained that PHP and JS/TS are programming languages, which contain translation functions such as `__()` and `_n()`, while HTML is a markup language, that does not contain translation functions.